CONTRIBUTING.md

```diff
@@ -156,6 +156,69 @@ pip install -r requirements-devel.txt
 and you will need to install recent git-annex using appropriate for your
 OS means (for Debian/Ubuntu, once again, just use NeuroDebian).
 
+Contributor Files History
+-------------------------
+
+The original repository provided a [.zenodo.json](.zenodo.json)
+file, and we generate a [.contributors file](.all-contributorsrc) from that via:
+
+```bash
+pip install tributors
+tributors --version
+0.0.18
+```
+
+If you want to have Orcid identifiers, you need to export the following environment
+variables. It also helps to have a GitHub token to increase API limits:
+
+```bash
+export ORCID_ID=xxxxxxxxxx
+export ORCID_SECRET=xxxxxxxxxxxxxxxxxx
+export ORCID_TOKEN=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+export ORCID_REFRESH_TOKEN=xxxxxxxxxxxxxxxxxxxxxxxxx
```

Comment:
I have some vague memory ORCID stuff isn't needed any longer, or only some of them?

Reply:
Yes I think that is correct!